### PR TITLE
Update SQL Script and Required Permissions

### PR DESCRIPTION
**Pull request by DCMattyG**
-Updated Arc-enabled SQL server onboarding script to match updates from ArcBox
-Adjusted required Service Principal permissions in documentation to "Owner"

### CODE REVIEW

**Comment by DCMattyG** [comment]
fixes #1394

File: azure_arc_sqlsrv_jumpstart/azure/arm_template/scripts/installArcAgentSQL.ps1 [hunk 1]
```diff
@@ -4,10 +4,9 @@
     [string]$servicePrincipalTenantId
 )
 
-$Env:JumpstartLogsDir = "C:\Jumpstart\Logs"
-$Env:JumpstartTempDir = "C:\Temp"
+$ArcBoxLogsDir = "C:\ArcBox\Logs"
```
Comment (dkirby-ms) [inline azure_arc_sqlsrv_jumpstart/azure/arm_template/scripts/installArcAgentSQL.ps1:7]: ArcBox?

File: azure_arc_sqlsrv_jumpstart/azure/arm_template/scripts/installArcAgentSQL.ps1 [hunk 2]
```diff
@@ -4,10 +4,9 @@
...
 
-Start-Transcript -Path $Env:JumpstartLogsDir\installArcAgentSQL.log
+Start-Transcript -Path $ArcBoxLogsDir\installArcAgentSQL.log
```
Comment (dkirby-ms) [inline azure_arc_sqlsrv_jumpstart/azure/arm_template/scripts/installArcAgentSQL.ps1:9]: ArcBox?

File: azure_arc_sqlsrv_jumpstart/azure/arm_template/scripts/installArcAgentSQL.ps1 [hunk 3]
```diff
@@ -17,7 +16,7 @@ $resourceGroup = $myResourceGroup
 $location = $azureLocation
 $proxy=""
 $resourceTags= @{"Project"="jumpstart_arcbox"}
-$arcMachineName = $Env:COMPUTERNAME
+$arcMachineName = "ArcBox-SQL"
```
Comment (dkirby-ms) [inline azure_arc_sqlsrv_jumpstart/azure/arm_template/scripts/installArcAgentSQL.ps1:19]: We dont need no stinking ArcBox?

**Review by dkirby-ms — CHANGES_REQUESTED** [review]
Some ArcBox logging paths in this non-ArcBox scenario.

File: azure_arc_sqlsrv_jumpstart/azure/arm_template/scripts/installArcAgentSQL.ps1 (hunk 1 above)
Comment (DCMattyG) [reply azure_arc_sqlsrv_jumpstart/azure/arm_template/scripts/installArcAgentSQL.ps1]: Pushed updated script code, resolved.

File: azure_arc_sqlsrv_jumpstart/azure/arm_template/scripts/installArcAgentSQL.ps1 (hunk 3 above)
Comment (DCMattyG) [reply azure_arc_sqlsrv_jumpstart/azure/arm_template/scripts/installArcAgentSQL.ps1]: Pushed updated script code, resolved.

File: azure_arc_sqlsrv_jumpstart/azure/arm_template/scripts/installArcAgentSQL.ps1 (hunk 2 above)
Comment (DCMattyG) [reply azure_arc_sqlsrv_jumpstart/azure/arm_template/scripts/installArcAgentSQL.ps1]: Pushed updated script code, resolved.

**Review by dkirby-ms — APPROVED** [review]

**Pull request merged**